source/macos/AdaptiveCards/AdaptiveCards/Views/ACRTextField.swift

```diff
@@ -85,6 +127,12 @@ class ACRTextField: NSTextField {
     private func updateClearButton() {
         clearButton.isHidden = isEmpty
     }
+    
+    private var wantsClearButton: Bool {
+        get {
+            return config.wantsClearButton || isDateTimeMode
+        }
```

Comment:
```suggestion
      return config.wantsClearButton || isDateTimeMode
```
`get` can be ignored, if only getter is used.